### PR TITLE
WPF-61657 Container Header support document is added

### PR DESCRIPTION
**Pull request by KarkuvelRajan**
WPF-61657 Container Header support document is added

### CODE REVIEW

File: wpf/Diagram/Container.md [hunk 1]
```diff
@@ -99,6 +99,112 @@ You can interactively add or remove diagram elements from the Container in the r
 
 ![WPF Diagram Container Interaction](Container_images/Container.gif)
 
+## Header support in Container
```
Comment (prakashperumal1892) [inline wpf/Diagram/Container.md:102]: Setting a Header

File: wpf/Diagram/Container.md [hunk 2]
```diff
@@ -99,6 +99,112 @@ You can interactively add or remove diagram elements from the Container in the r
...
+
+* The `Header` property of container allows you to textually describe the container and to customize the appearance of the description.
+* The size of the container header can be controlled by using the `UnitWidth` and `UnitHeight` properties of header.
```
Comment (prakashperumal1892) [inline wpf/Diagram/Container.md:105]: You can textually describe a container by using its 'Header' property. Also, users can customize the header's appearance using header's "ShapeStyle" property.

The following code example explains how to define a header for the container and alter its appearance.




You can edit the header by double-clicking the region of the container's header or by pressing F2.

**Review by prakashperumal1892 — CHANGES_REQUESTED** [review]

File: wpf/Diagram/Container.md [hunk 3]
```diff
@@ -87,6 +87,119 @@ diagram.Groups = groups;
 
 [View sample in GitHub](https://github.com/SyncfusionExamples/WPF-Diagram-Examples/tree/master/Samples/Container)
 
+### Setting a Header
+
+You can textually describe a container by using its 'Header' property. Also, users can customize the header's appearance using header's "ShapeStyle" property.
```
Comment (sarathkumarvenkatesan) [inline wpf/Diagram/Container.md:92]: Link property to its API link

Comment (KarkuvelRajan) [reply wpf/Diagram/Container.md:92]: Hi @sarathkumarvenkatesan 

After this Volume release only we will be able to add the API links.

Regards,
Karkuvel Rajan S

Comment (OnyimbiNelson) [inline wpf/Diagram/Container.md:92]: You can textually describe a container by using its 'Header' property. Also, users can customize the header's appearance using the header's "ShapeStyle" property.

File: wpf/Diagram/Container.md [hunk 4]
```diff
@@ -87,6 +87,119 @@ diagram.Groups = groups;
...
+
+The following code example explains how to define a container header and its customization.
```
Comment (OnyimbiNelson) [inline wpf/Diagram/Container.md:94]: The following code example explains how to define a container header and its customization:

File: wpf/Diagram/Container.md [hunk 5]
```diff
@@ -87,6 +87,119 @@ diagram.Groups = groups;
...
+    },
+};
+
+groups.Add(container);
+diagram.Groups = groups;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> You can edit the header by double-clicking the region of the container's header or by pressing F2.
```
Comment (OnyimbiNelson) [inline wpf/Diagram/Container.md:199]: N> You can edit the header by double-clicking the region of the container's header, or by pressing F2.

File: wpf/Diagram/Container.md [hunk 6]
```diff
@@ -87,6 +87,119 @@ diagram.Groups = groups;
...
+
+![WPF Diagram Container Header](Container_images/ContainerHeader.png)
+
 ### Container from Stencil
 
 Container Nodes can be predefined and added to stencil. You can drop those Containers into Diagram, when required.
```
Comment (OnyimbiNelson) [inline wpf/Diagram/Container.md:205]: Container Nodes can be predefined and added to the stencil. You can drop those Containers into Diagram when required.

**Review by sarathkumarvenkatesan — APPROVED** [review]

**Review by sarathkumarvenkatesan — APPROVED** [review]

**Review by prakashperumal1892 — APPROVED** [review]
Approved

**Pull request merged**